package.json

```diff
@@ -25,29 +25,29 @@
     "typecov": "type-coverage"
   },
   "devDependencies": {
-    "@1stg/lib-config": "^1.2.10",
-    "@1stg/tslint-config": "^1.2.0",
-    "@types/eslint": "^7.2.10",
+    "@1stg/lib-config": "^3.0.0",
+    "@1stg/tslint-config": "^2.0.0",
+    "@types/eslint": "^7.2.14",
     "@types/eslint-plugin-markdown": "^2.0.0",
-    "@types/jest": "^26.0.23",
-    "@types/node": "^14.14.41",
-    "@types/react": "^17.0.4",
-    "@types/unist": "^2.0.3",
+    "@types/jest": "^26.0.24",
+    "@types/node": "^16.3.0",
+    "@types/react": "^17.0.14",
+    "@types/unist": "^2.0.5",
```

Comment:
I find it a little strange bumping package.json for in range versions unless there is something semver breaking that was added in a non-semver major release.
That said, for dev dependencies, it probably doesn't matter :slightly_smiling_face:

Reply:
I want to always follow the latest versions.

`@types/unist` has a *breaking change* at https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54181

Reply:
I want to always follow the latest versions.

`@types/unist` has a *breaking change* at https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54181

Reply:
> I want to always follow the latest versions.

right, and `^` and `~` can also do that to a large extend :slightly_smiling_face: 

> `@types/unist` have a breaking change at DefinitelyTyped/DefinitelyTyped#54181

Yeah, that one is a bit tricky semver wise.
`@types/unist` follows the unist standard https://github.com/syntax-tree/unist/releases in terms of major/minor version numbering, here a fix which is potentially breaking in some situations was brought in to fix some type narrowing issues.

Reply:
> right, and ^ and ~ can also do that to a large extend 🙂

We have `yarn.lock`, so I personally still prefer update `package.json` at the same time and keep `yarn.lock`.